Core/MaterialParameters.cpp

```diff
@@ -60,6 +60,22 @@ const MaterialConstants MaterialParameters::fAir = MaterialConstants(
   {142 * ns}, {75}, 0.125 * ns, 1.0 / 3.0, {0.5 * ns}, {0}
 );
 
+const MaterialConstants MaterialParameters::fPolycarbonate = MaterialConstants( 
+  {3 * ns}, {25}, 0.125 * ns, 1.0/3.0, {0.3 * ns}, {100}
+);
+
+const MaterialConstants MaterialParameters::fPolyoxymethylene = MaterialConstants(
+  {1.6 * ns}, {20}, 0.125 * ns, 1.0/3.0, {0.3 * ns}, {100}
+);
+
+const MaterialConstants MaterialParameters::fSiliconDioxide = MaterialConstants( 
+  {2 * ns}, {30}, 0.125 * ns, 1.0/3.0, {0.35 * ns}, {100}
+);
+
+const MaterialConstants MaterialParameters::fStainlessSteel = MaterialConstants(
```

Comment:
Some info what kind of steel it was? serial number or anything?
Artciles:
doi:10.1016/j.fusengdes.2017.04.116
https://link.springer.com/article/10.1007/s11249-011-9890-7